### PR TITLE
Fixes #18 - Addresses potential architecture on geoplatform.gov

### LINKED ISSUE #18: Decision: As a developer, I want to understand if geoplatform.gov is the best option for data storage for J40.
**Description**
Now that we have access, we should investigate the geoplatform.gov feature set and determine if it can host our datasets, layers, services, and/or FE code. 

**Solution**
Create ADR for using Geoplatform based on research from team.

**Describe alternatives you've considered**
* Cloud.gov
* Federalist
* ArcGIS Hub

**Links to user research or other resources**
Research to be included in ADR

**Tasks**
- [ ] Meet with Joel @ Geoplatform to understand what abilities our credentials grant us
- [ ] Discuss with him and document available feature sets
- [ ] Identify opportunities for dataset hosting
- [ ] Identify opportunities for layer hosting
- [ ] Identify opportunities for static site hosting
- [ ] Identify opportunities for mvt/tile service hosting 

**Definition of "Done"**
- [ ] We've compiled an ADR that includes the above sections and compares the available options

### PR DESCRIPTION
**Pull request by NatHillardUSDS**
This is a tentative proposed architecture for our interface with Geoplatform.gov. Setting this as `Draft` for now, as we are still in discussions, but we wanted to make this public as a point of discussion and iteration.

Addresses:
* https://github.com/usds/justice40-tool/issues/18 - "As a developer, I want to understand if geoplatform.gov is the best option for data storage for J40. "
  * We are evolving how this will work in conversation with Geoplatform directly
* https://github.com/usds/justice40-tool/issues/36 - "As a developer, I want to understand and document tile hosting options, so that I can make an informed decision about how to vend map data" 
  * If we do go forward with Geoplatform, according to this proposed architecture, we will not have to roll our own tileserver, which has a lot of benefits that that ticket also goes over.

### CODE REVIEW

File: docs/decisions/0005-hosting.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Technical Story: https://github.com/usds/justice40-tool/issues/18 + https://github.com/usds/justice40-tool/issues/36 
+
+## Context and Problem Statement
+We need a host for our data pipeline and front end website/app.
+
+## Decision Drivers
+
+- **Speed of launch** - We have a mandate to launch an initial version of our tool by July 27, and we want to get an informational site up much sooner.
+- **Support for continuous integration and delivery** - We need our hosting provider to support a modern software development lifecycle that includes continuous integration and delivery, such as integration with our Github repository and CI/CD tool such as Github Actions.
+- **Ease of implementation** -
```
Comment (NatHillardUSDS) [inline docs/decisions/0005-hosting.md:17]: Small nit but this point is empty at the moment

File: docs/decisions/0005-hosting.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+
+## Considered Options
+
+- Geoplatform
+- Cloud.gov
+
+## Decision Outcome
+
+We will use Geoplatform.gov for hosting our data pipeline, tile server, and front end client. They have a set of shared service offerings that will enable us to have an open data pipeline from data source to tile API, enabling contributions to data processing at any point as well as enabling data access for users at any point (e.g. whether a user wants to access raw data or GeoJSON or tile format). See this diagram that illustrates the proposed system architecture and hosting: docs/architecture/architecture.mmd
```
Comment (NatHillardUSDS) [inline docs/decisions/0005-hosting.md:28]: You can link to the diagram with ```diagram```

File: docs/decisions/0005-hosting.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Pros:
+- Control over software development lifecycle including open source code and CI/CD setup
+- Supports CI/CD
+- Supports a shared service in government
+- The team has experience with Cloud.gov
+
+Cons:
+- We would not be able to launch our website or backend on our expected timeline
+- Would require a lot more build on our part for the data pipeline and tile server
+
```
Comment (NatHillardUSDS) [inline docs/decisions/0005-hosting.md:54]: I would also add here `- Would not have the flexibility to easily run serverless functions if we needed to`

**Review by NatHillardUSDS — COMMENTED** [review]
Looks great overall! A few small changes but otherwise good to go - excited about this!

**Review by switzersc-usds — APPROVED** [review]
Updated per feedback!

**Comment by NatHillardUSDS** [comment]
LGTM! (As I opened the PR for the diagram I can’t approve directly but Shelby’s changes look good!)

**Pull request merged**